### ISSUE TITLE
XLA Tests are broken

### ISSUE THREAD

**Issue by zengk95** [pytorch/pytorch#76243]
> NOTE: Remember to label this issue with "`ci: sev`"

## Current Status
mitigated

## Error looks like
```
2022-04-22T16:16:17.8924985Z [1A[K[31m[1mFAILED:[0m Build did NOT complete successfully
2022-04-22T16:16:17.8999989Z 
2022-04-22T16:16:17.9003287Z [1A[K[31m[1mFAILED:[0m Build did NOT complete successfully
2022-04-22T16:16:17.9115269Z [0mFailed to build external libraries: ['/var/lib/jenkins/workspace/xla/build_torch_xla_libs.sh', '-O', '-D_GLIBCXX_USE_CXX11_ABI=1', 'install']
```
## Incident timeline (all times pacific)
-  April 21st 12:30 PM: #75324 was landed and author notified us that XLA was broken but would be fixed later on
- April 22nd 9:AM: #76237 was landed to disable this test due to XLA fix still not landed. Will bring it back once XLA fix is finished.

## User impact
Master CI is broken and any subsequent PRs based on that will be broken.

## Root cause
A PR that is dependent on future XLA changes.

## Mitigation
Disabled the test 

## Prevention/followups
Make sure forward fix is ready to go before merging in the breaking PR.

**Comment by seemethere** [comment]
cc @JackCaoG

**Comment by JackCaoG** [comment]
I just reverted the pr https://github.com/pytorch/xla/pull/3522. It is due to some cache issue from bazel

**Comment by JackCaoG** [comment]
We update the tensorflow pin in one of yesterday prs, recent TF has a known Bazel compilation issue if the cache is not clean. I think for us to proceed we need to disable the remote cache on upstream pt/xla build for a couple days at least.

**Comment by zengk95** [comment]
Closing this issue for now since we disabled it for mitigation.

**Comment by JackCaoG** [comment]
@zengk95 Can we reenable the pt/xla CI as the offending pr on pt/xla side is reverted?

**Comment by zengk95** [comment]
> > NOTE: Remember to label this issue with "`ci: sev`"
> 
> ## Current Status
> mitigated
> 
> ## Error looks like
> ```
> 2022-04-22T16:16:17.8924985Z �[1A�[K�[31m�[1mFAILED:�[0m Build did NOT complete successfully
> 2022-04-22T16:16:17.8999989Z 
> 2022-04-22T16:16:17.9003287Z �[1A�[K�[31m�[1mFAILED:�[0m Build did NOT complete successfully
> 2022-04-22T16:16:17.9115269Z �[0mFailed to build external libraries: ['/var/lib/jenkins/workspace/xla/build_torch_xla_libs.sh', '-O', '-D_GLIBCXX_USE_CXX11_ABI=1', 'install']
> ```
> 
> ## Incident timeline (all times pacific)
> * April 21st 12:30 PM: [Decouple Lazy Node Shape Cache #75324](https://github.com/pytorch/pytorch/pull/75324) was landed and author notified us that XLA was broken but would be fixed later on
> * April 22nd 9:AM: [disable xla test job #76237](https://github.com/pytorch/pytorch/pull/76237) was landed to disable this test due to XLA fix still not landed. Will bring it back once XLA fix is finished.
> 
> ## User impact
> Master CI is broken and any subsequent PRs based on that will be broken.
> 
> ## Root cause
> A PR that is dependent on future XLA changes.
> 
> ## Mitigation
> Disabled the test
> 
> ## Prevention/followups
> Make sure forward fix is ready to go before merging in the breaking PR.

Yes I will do that. Sorry, I missed this!

**Comment by JackCaoG** [comment]
Thanks @zengk95 , @yeounoh is verifying upstream test is passing on our current master. We can ping you again when we think it is safe to turn on test again.

**Comment by zengk95** [comment]
> Thanks @zengk95 , @yeounoh is verifying upstream test is passing on our current master. We can ping you again when we think it is safe to turn on test again.

So it's not safe to turn it back on? Can you accept this #76403 when it's fixed? We may be losing signal on this test the longer it's disabled and if there's a PR that breaks it while it's disabled, it may take a while to bisect it.
